pages/base.page.js

```diff
@@ -0,0 +1,44 @@
+const PAGE_TIMEOUT = 10000
+export default class BasePage{
+    
+    /**
+     * Abrir pagina
```

Comment:
These comments need to be in english as well. Translate them on every element you have them.